.env

```diff
@@ -0,0 +1,9 @@
+MYSQL_DATABASE=BookShop
+MYSQL_USER=appuser               # Added a non-root user
+MYSQL_PASSWORD=qwertyasd
+MYSQL_ROOT_PASSWORD=rootpassword
+MYSQL_LOCAL_PORT=3307
+MYSQL_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8080
+SPRING_DOCKER_PORT=8080
```

Comment:
Don’t push files with sensitive information. Add it to .gitignore. Better add .env.template without values